bin/console

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env ruby
 
-require "bundler/setup"
+$: << File.join(File.dirname(__FILE__), '..', 'lib')
 
-# You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
+require "console/console"
+require "core/players/players"
+require "core/players/player_symbol"
+require "core/board"
+require "core/game"
 
-# (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
-# Pry.start
+console = Console.new(input: $stdin, output: $stdout)
```

Comment:
I'd be tempted to hide these as defaults inside the class constructor with something like:
`@input = options[:input] || $stdin`.